app/views/layouts/_footer.haml

```diff
@@ -26,6 +24,5 @@
     %ul.list-inline
       %li
         %span#cdSiteSeal1
-          %script{ :src =>
-            "//tracedseals.starfieldtech.com/siteseal/get?scriptId=cdSiteSeal1&cdSealType=Seal1&sealId=55e4ye7y7mb730aa1fd9889fd9470d76zzxxy7mb7355e4ye74ffe7b91ae2b110",
+          %script{ :src => "//tracedseals.starfieldtech.com/siteseal/get?scriptId=cdSiteSeal1&cdSealType=Seal1&sealId=55e4ye7y7mb730aa1fd9889fd9470d76zzxxy7mb7355e4ye74ffe7b91ae2b110",
```

Comment:
Line is too long. [184/80]<br>Hash attribute should end with one space before the closing brace